package.json

```diff
@@ -6,6 +6,9 @@
   "directories": {
     "doc": "docs"
   },
+  "eslintConfig": {
```

Comment:
From my understanding of the [eslint docs](https://eslint.org/docs/user-guide/configuring), we specify the configuration either in `package.json`, or in `.eslintrc.js`. Is there a reason you're adding this here instead of in the `.eslintrc.js`?

Reply:
I likely picked that redundancy up as I cobbled together various package configurations. I will review.